roles/ovirt-mac-pools/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: set mac pool
+  ovirt_mac_pools:
+    auth: "{{ ovirt_auth }}"
+    name: "{{ mac_pool_name | default('Default') }}"
```

Comment:
Do we want to always by default create a 'Default' mac pool range? I don't think it's good idea.